controllers/machine_controller.go

```diff
@@ -92,6 +122,24 @@ func (r *MachineReconciler) SetupWithManager(mgr ctrl.Manager, options controlle
 	return nil
 }
 
+func (r *MachineReconciler) reconcileRequests(a handler.MapObject) []reconcile.Request {
+	requests := []reconcile.Request{}
+	machines, err := getActiveMachinesInCluster(context.Background(), r.Client, a.Meta.GetNamespace(), a.Meta.GetName())
```

Comment:
I'm not sure this makes much difference, but I think we've been using context.Background in main.go and .TODO everywhere else. I realize we may never be able to plumb a context in here, so 🤷‍♂. @vincepri do you have a strong preference?

Reply:
both are functionally the same, their meaning is different though. TODO should be used if we're planning to add the context chain at some point. As you pointed out though, we'll probably not going to be able to do that here, so it's probably ok to keep Background